src/components/backend-ai-storage-list.ts

```diff
@@ -2101,6 +2108,11 @@ export default class BackendAiStorageList extends BackendAIPage {
    * @param {Event} e - click the
    */
   _renameFolderDialog(e) {
+    this.renameFolderID = globalThis.backendaiclient.supports(
+      'vfolder-id-based',
+    )
+      ? this._getControlID(e)
+      : this._getControlName(e);
```

Comment:
Wouldn't it be better to have the folderId determined by the Vfolder class in ems and have the ID in that variable? (like 2803 line)

Reply:
It is updated when only calling `_updateFolderName`.